core/app/(user)/forgot/ForgotForm.tsx

```diff
@@ -21,6 +21,10 @@ const forgotPasswordSchema = z.object({
 export default function ForgotForm() {
 	const form = useForm<z.infer<typeof forgotPasswordSchema>>({
 		resolver: zodResolver(forgotPasswordSchema),
+		defaultValues: {
+			// in order to prevent "Form changed from uncontrolled to controlled" React errors
+			email: "",
+		},
```

Comment:
i added a number of these things in order to get rid of some errors, which were making tests fail in dev mode bc the error popup would overlap with the loginswitcher 😅